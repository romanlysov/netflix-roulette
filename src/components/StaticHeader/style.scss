.navigation {
  display: flex;
  flex-wrap: wrap;
  flex-direction: row;
  justify-content: space-between;
  align-items: flex-start;
  padding: 0 6.25rem;
}
