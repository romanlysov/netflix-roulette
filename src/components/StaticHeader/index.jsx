import React from 'react'

import { MainLogo } from '../MainLogo'
import { SearchButton } from '../SearchButton'

import './style.scss'

export const StaticHeader = ({ className, onclick, searchButtonClass }) => {
  return (
    <nav className={`header__static ${className}`}>
      <MainLogo className="navigation__logo" />
      <SearchButton
        classAttribute={searchButtonClass}
        value="search"
        action={onclick}
      />
    </nav>
  )
}
