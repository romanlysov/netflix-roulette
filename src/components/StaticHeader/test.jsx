import React from 'react'
import renderer from 'react-test-renderer'

import { StaticHeader } from 'components/FilmHeader/StaticHeader/index'

test('renders StaticHeader properly', () => {
  const wrapper = renderer.create(<StaticHeader />).toJSON()
  expect(wrapper).toMatchSnapshot()
})
