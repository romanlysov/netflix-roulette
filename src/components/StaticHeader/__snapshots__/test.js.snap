// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders StaticHeader properly 1`] = `
<nav
  className="header__static undefined"
>
  <img
    alt=""
    className="navigation__logo"
    src="test-file-stub"
  />
  <button
    className="navigation__search"
  >
    search
  </button>
</nav>
`;
